Dockerfile

```diff
@@ -16,6 +16,5 @@ WORKDIR /workbench
 RUN python -m pip install setuptools
```

Comment:
Setuptools may become essential on Py3.12 and beyond because distutils was removed from the Standard Library.